content/docs/user-guide/data-management/remote-storage/hdfs.md

```diff
@@ -47,6 +47,14 @@ them with the `--local` option, so they're written to a Git-ignored config file.
                               kerb_ticket /path/to/ticket/cache
   ```
 
+- `replication` - replication factor for write operations on HDFS cluster.
+  default value is 3.
+
+  ```cli
+  $ dvc remote modify myremote \
```

Comment:
can we now just keep it on a single line? (remove `\`)

Reply:
Fixed.